### ISSUE TITLE
no report when using cucumber, examples and not tags (!)

### ISSUE THREAD

**Issue by asvdw** [serenity-bdd/serenity-core#1092]
Hello,
Since a few releases I have a strange problem generating reports after tests using cucumber (jvm). That's ok using serenity 1.5.11 and serenity.cucumber 1.5.13, but with any newer versions tests run fine but aggregate says something like:
```
[INFO] --- serenity-maven-plugin:1.8.3:aggregate (serenity-reports) @ serenityLab ---
[INFO] current_project.base.dir: /Users/me/Documents/projets/labs/serenity/serenityLab
[INFO] Generating test results for 0 tests
[INFO] 3 requirements loaded after 384 ms
[INFO] 3 related requirements found after 384 ms
[INFO] Generating test outcome reports: false
[INFO] Starting generating reports: 408 ms
[INFO] Configured report threads: 20
[INFO] Test results for 0 tests generated in 1466 ms
[INFO] 
```
Then of course reports are empty
I was finally able to reproduce it, that was not easy !
**Steps**
* mvn archetype:generate, select Serenity jvm, latests versions (1.8.3 - cucumber 1.6.6)
* run tests (mvn clean verify), ok
* update LookupADefinition like this. It's required to use data table.
```
Feature: Lookup a definition
  In order to talk better
  As an English student
  I want to look up word definitions

  Scenario: Looking up the definition of 'apple'
    Given the user is on the Wikionary home page
    When the user looks up the definition of the word 'apple'
    Then they should see the definition 'A common, round fruit produced by the tree Malus domestica, cultivated in temperate climates.'

    @NotThisOne
  Scenario: Looking up the definition of pear
    Given the user is on the Wikionary home page
    When the user looks up the definition of the word 'pear'
    Then they should see the definition 'An edible fruit produced by the pear tree, similar to an apple but elongated towards the stem.'

  Scenario Outline: Looking up the definition of fruits
    Given the user is on the Wikionary home page
    When the user looks up the definition of the word '<name>'
    Then they should see the definition 'An edible fruit produced by the pear tree, similar to an apple but elongated towards the stem.'
    Examples:
    | name |
    | pear |
```
* run tests, all is ok
* Now update runner DefinitionTestSuite.java:
```

@RunWith(CucumberWithSerenity.class)
@CucumberOptions(features="src/test/resources/features/consult_dictionary/LookupADefinition.feature", tags={"@NotThisOne"})
public class DefinitionTestSuite {
```
** still ok, wonderful
* last step, now change tags option to use NOT keyword. You can use both old and new syntax: "~@NotThisOne" or "not @NotThisOne":
```

@RunWith(CucumberWithSerenity.class)
@CucumberOptions(features="src/test/resources/features/consult_dictionary/LookupADefinition.feature", tags={"~@NotThisOne"})
public class DefinitionTestSuite {}
```
* Run tests again... tests run fine but report is empty !

It takes me hours to fine the "right" combinaison, so I have not tried to resolve it up to now; any idea ?
Thank you !

**Comment by cliviu** [comment]
hi @asvdw , can you please provide a small project to help us to reproduce the issue ?

**Comment by asvdw** [comment]
yes...
https://github.com/asvdw/serenity

**Comment by cliviu** [comment]
thanks for your help, I will analyse the issue

**Comment by cliviu** [comment]
@asvdw , thanks for spotting the issue

**Comment by kutzi** [comment]
Thanks for the fix. When can we expect a new release containing the fix?

**Comment by asvdw** [comment]
Thank you, I'll have a look asap !

**Comment by wakaleo** [comment]
Fixed in 1.8.x

**Comment by asvdw** [comment]
I still have to test "in real life" but my tests now pass.  Thank you !
